FWCore/Utilities/scripts/edmCheckClassTransients

```diff
@@ -34,25 +34,26 @@ class RMParser(object):
           f.close()
 
 def checkTrans(templname,name):
-    print "Checking "+name
     c = ROOT.TClass.GetClass(name)
     if not c:
-        print "failed to load dictionary for class '"+name+"'. Check for typedefs in the name."
-        return
+        print "Info: Could no load dictionary for class '"+name+"' because of typedef(s) in the name."
+        return 0
     for trans in classtransients[templname]:
       tdm = c.GetDataMember(trans)
       retval = False
       if tdm : retval = tdm.IsPersistent()
-      if retval == True : raise RuntimeError("Error: field '"+trans+"' must be labeled transient=\"true\" in classes_def.xml for '"+name+"'")
-
+      if retval == True : 
+          print "Error: field='"+trans+"' must be labeled transient=\"true\" in classes_def.xml for "+name+" or the equivalent class name with typedefs."
+          return 1
```

Comment:
why return here? don't you want to keep looping for the other tansients classes?